core/core.go

```diff
@@ -119,9 +119,19 @@ var ErrStarted = fmt.Errorf("node is started")
 var ErrStopped = fmt.Errorf("node is stopped")
 var ErrOffline = fmt.Errorf("node is offline")
 
+// RepoPath returns the actual location of the configured repo
+func (conf InitConfig) RepoPath() string {
+	return path.Join(conf.BaseRepoPath, conf.Account.Seed())
+}
+
+// RepoExists return whether or not the configured repo already exists
+func (conf InitConfig) RepoExists() bool {
+	return fsrepo.IsInitialized(conf.RepoPath())
+}
+
```

Comment:
These methods for `InitConfig` provide a way to get the full repo path (base path + seed) and check if the repo is initialized already. Equivalent methods exist in the mobile module.